src/copyCodeBlock.js

```diff
@@ -2,31 +2,57 @@ import { getDisplayString, getClipboardString } from './utils';
 import styles from './styles';
 
 export default (string, opts = {}) => {
-    const lang = opts && opts.lang || '';
+    const { lang = 'HTML', shouldReturnDomEl = false } = opts;
```

Comment:
We shouldn't be defaulting `lang`:
* It'll always produce a warning if `hightlight.js` isn't installed
* We shouldn't try to syntax highlight if no language is supplied

Reply:
`lang` is only being used for the `aria-label`, it's not mutating `opts.lang`.

That being said, you're still right. I don't want the screen reader to say "undefined code block"